pages/register.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { register } from "../store/auth/slice";
+
+import Router from "next/router";
+
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import { selectRegisterError } from "../store/auth/selectors";
+
+import Container from "react-bootstrap/Container";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+
+import styles from '../styles/App.module.css'
+
+const validationSchema = Yup.object({
```

Comment:
Put the validation schema in it's own file, could be named validation.js, and make sure it's not in the pages folder.